_posts/2020-3-29-managed-services-serverless.md

```diff
@@ -0,0 +1,187 @@
+  forProvider:
+    databaseVersion: POSTGRES_11
+    region: us-west2
+    settings:
+      tier: db-custom-1-3840
+      dataDiskType: PD_SSD
+      dataDiskSizeGb: 10
+      ipConfiguration:
+        ipv4Enabled: true
+        authorizedNetworks:
+          - value: "0.0.0.0/0" # whitelist all IPs so we can access from local machine
+  providerRef:
+    name: gcp-provider
+  reclaimPolicy: Delete
+```
+
+As you can see, a Postgres database is a bit more complicated than an object storage bucket, but this level of detail is once again confined to the infrastructure operator, while the developer simply creates a `PostgreSQLInstance` claim that can be satisfied by the `CloudSQLInstanceClass`. The contents of the claim are significantly less verbose:
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: PostgreSQLInstance
+metadata:
+  name: comment-vibes-db
+  namespace: openfaas-fn
+spec:
+  classSelector:
+    matchLabels:
+      app: comment-vibes
+  writeConnectionSecretToRef:
+    name: pgconn
+  engineVersion: "11"
+```
+
+After creating both the `CloudSQLInstanceClass` and the `PostgreSQLInstance` claim, GCP began to provision the database. This [was reflected](https://youtu.be/XphQgB87U-s?t=3190) in the state of the `CloudSQLInstance` object in our Kubernetes cluster, as it reported back `PENDING_CREATE`:
+
+![GCP Pending Create](/images/2020-03-30-managed-services-serverless/gcp-pending-create.png)
+
+After a short period of time, the Postgres database was fully provisioned and the connection information was propagated to the `openfaas-fn` namespace in the form of a Kubernetes `Secret`. We used the connection information to [setup our database and table schema](https://youtu.be/XphQgB87U-s?t=3628) prior to deploying our functions.
+
+```console
+PGPASSWORD=$(echo $PASSWORD) psql -h $HOST -p 5432 -U $USERNAME -c 'CREATE DATABASE defaultdb;'
+PGPASSWORD=$(echo $PASSWORD) psql -h $HOST -p 5432 -U $USERNAME defaultdb < schema.sql
+```
+
+![Seed Postgres Database](/images/2020-03-30-managed-services-serverless/seed-db.png)
+
+Now our functions were ready to consume the Postgres database, so we deployed them using `faas-cli deploy`. The next step was relying on inlets again to publicly expose our endpoint so that Github could the `import-comments` function as a webhook to send events. Alex set up the webhook and [Github issue](https://github.com/teamserverless/proposals/issues/3) while I exposed the endpoints using the inlets client.
+
+![Inlets Client](/images/2020-03-30-managed-services-serverless/inlets-client.png)
+
+We finished up by sharing the issue link the the live stream audience, then [watched to see](https://youtu.be/XphQgB87U-s?t=3957) which emoji rose to the top.
+
+![View Emojis](/images/2020-03-30-managed-services-serverless/view-emojis.png)
+
+# Watch the Live Stream
+
+If you enjoyed these demos, watch the full live stream to see them in action and learn more about OpenFaaS and Crossplane.
```

Comment:
You could also mention that you interviewed me on openfaas and note one or two questions you asked.

Something like: "You'll also hear me interview Alex at the start of the session where I ask questions like... " Why? It's a more engaging reason to click than "watch the full version"